main.go

```diff
@@ -18,6 +19,7 @@ import (
 
 const (
 	pathEnv = "PARAMETER_STORE_EXEC_PATH"
+	disableTranslation = "PARAMETER_STORE_EXEC_DISABLE_TRANSLATION"
```

Comment:
I try to avoid “disable = true”.
Perhaps `PARAMETER_STORE_EXEC_VERBATIM=true`?
Or perhaps `PARAMETER_STORE_EXEC_TRANSFORM=none`, which leaves room for the default `upper` and perhaps future transform modes (e.g. maybe one that doesn't trim the prefix at all).